osidb/tests/test_flaw.py

```diff
@@ -791,3 +791,38 @@ def test_validate_future_unembargo_date(self):
         with freeze_time(future_dt):
             FlawFactory(unembargo_dt=future_dt, embargoed=True)
             # no exception should be raised now
+
+    @pytest.mark.parametrize(
+        "is_embargoed,summary,should_raise",
+        [
+            (False, "Valid summary", False),
+            (False, "EMBARGOED - Invalid summary", True),
```

Comment:
nitpick: when `is_embargoed` is always `False` there is no reason to parametrize it